website/docs/components/badge/badge-a11y.md

```diff
@@ -9,4 +9,4 @@ Don’t hide the badges from assistive technology users with the `aria-hidden` a
 
 ## Other recommendations
 
-See more accessibility recommendations in the common [Accessibility guide](/core-principles/a11y/a11y).
+Read more accessibility recommendations in the common [Accessibility guide](/core-principles/a11y/a11y).
```

Comment:
What do you think about keeping this text the same throughout the documentation?
```suggestion
For more accessibility recommendations, refer to the common [Accessibility guide](/core-principles/a11y/a11y).
```